command/connect/envoy/envoy.go

```diff
@@ -24,6 +24,12 @@ import (
 )
 
 func New(ui cli.Ui) *cmd {
+	if bui, ok := ui.(*cli.BasicUi); ok {
+		// Prevent any incidental use of Output/Info from writing to Stdout and
+		// corrupting the generated bootstrap file.
+		bui.Writer = os.Stderr
```

Comment:
Wouldn't this mean that this log message would go to Stderr now?
https://github.com/hashicorp/consul/blob/master/command/connect/envoy/envoy.go#L383

That seems a bit odd

Reply:
One of the semi-"standard" things about log messages on unix is that informational messages go to stderr and only the real output of a program goes to stdout so it can be used in a pipeline. It certainly doesn't seem out of place.